lib/iris/util.py

```diff
@@ -849,18 +851,13 @@ def __eq__(self, other):
         return (isinstance(other, type(self)) and
                 self._identity() == other._identity())
 
-    def __ne__(self, other):
```

Comment:
We still need this for Python 2, otherwise the behaviour of `!=` will change. (`functools.total_ordering` doesn't help here.)